wLabs/LabSetup.md

```diff
@@ -0,0 +1,595 @@
+---
+title: Setting up the Lab Environment
+layout: custom
+keywords: [lab, guide, setup, start, FinCube, data cells]
+description: This page details the necessary requirements and steps to set up a local environment that can accommodate a user following the Lab Guide.
+---
+* * *
+
+## Overview
+
+The [Lab Guide](/wLabs/lab.html) provides a convenient way for users to become familiar with Interject. The Guide walks through several individual walkthroughs on how to modify and create reports, drilling to data, and exporting reports. In order for a user to follow along, it is necessary to first set up a local environment that can accommodate this Lab Guide. This page details the necessary requirements and steps to do this.
+
+### Interject Data Report Flow
+
+The reports in the Lab Guide contain [Interject report formulas](/wIndex/Excel-Function-Index.html), each referencing a [Data Portal](/wPortal/Data-Portals.html) setup on the [Interject Portal Website](/wPortal/The-INTERJECT-Website-Portal.html). These Data Portals in turn reference a Data Connection and a Stored Procedure that point to a data source where the data will be pulled from. When the data source is a database, the entire flow can be conceptualized by the following diagram:
+
+![](/images/LabSetup/FlowChart.png)
+<br>
+
+### Subscription Setup
+
+In order to have the necessary requirements to go through the Lab Guide, a user just needs to be subscribed to the "Publisher - Interject App". Once subscribed, a user can select this company from the Interject ribbon:
```

Comment:
Currently users cannot self-subscribe to an Interject Published App. We might want to consider providing a page or contact info on how to become subscribed to the Published App: "Publisher - Interject App".

Reply:
i didnt realize requesting a subscription is mentioned below. I would suggest moving the "how to get subscribed" details closer to the top here when published apps are first mentioned.

Reply:
Yes good point, I will move the request info to the top.